runtime/cudaq/platform/pasqal/PasqalServerHelper.cpp

```diff
@@ -52,50 +55,60 @@ RestHeaders PasqalServerHelper::getHeaders() {
 
 ServerJobPayload
 PasqalServerHelper::createJob(std::vector<KernelExecution> &circuitCodes) {
-  ServerJobPayload ret;
-  std::vector<ServerMessage> &tasks = std::get<2>(ret);
+  std::vector<ServerMessage> tasks;
+
   for (auto &circuitCode : circuitCodes) {
-    // Construct the job message
-    ServerMessage taskRequest;
+    ServerMessage message;
+    message["name"] = circuitCode.name;
+    message["device"] = backendConfig.at("machine");
+    message["shots"] = shots;
 
-    taskRequest["name"] = circuitCode.name;
-    taskRequest["device"] = backendConfig.at("machine");
     auto action = nlohmann::json::parse(circuitCode.code);
-    taskRequest["action"] = action.dump();
-    taskRequest["shots"] = shots;
+    message["action"] = action.dump();
```

Comment:
what is this again?

Reply:
The compiled code. Which in our case, should be `sequence`? I need to double check how it's being organised.

Reply:
The names don't seem to be standarized.

Reply:
If we look at QuEra's [test](https://github.com/awennersteen/cuda-quantum/blob/main/unittests/backends/quera/JsonPayloadTester.cpp), the `setup` and the `hamiltonian` are being passed as `action` (separated from the specifications and the number of shots). In our case, should we split it into two parts?

Reply:
that's ok, I've implemented it like in their test